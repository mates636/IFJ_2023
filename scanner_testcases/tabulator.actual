token type: 1 
token data: "a" 
token type: 19 
token type: 10 
token data: "ahoj" 
token type: 11 
