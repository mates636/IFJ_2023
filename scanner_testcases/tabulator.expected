1 "a"
15
7 "ahoj"
8
